### DISCUSSION TITLE
How to sort and download a single type of data ?

### DISCUSSION THREAD

**Discussion by 666diabolus** [UltimaHoarder/UltimaScraper#1294] [General]
For example if I only wanted to download only things in Messages/DM, what settings should I use ?

**Comment by UltimaHoarder** [comment]
auto_api_choice: "Messages" or replace Messages with the number you have to pick for messages. 
I need to document it properly

**Comment by 666diabolus** [comment]
Thanks for the answer @DIGITALCRIMINAL . And I got some questions : 
Can I run multiple scrapt at one time on a single account / single IP ? 
Can I run multiple scrapt at one time on multiple account in one single IP ?
